about.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+
+    <title>Tammy Fisher's Resume</title>
+
+    <link rel="stylesheet" href="css/index.css" />
+    <meta name="viewport" content="width=divice-width, initial-scale=1.0" />
+
+    <!--[if lt IE 9]>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
+  </head>
+
+  <body class="container">
+    <header>
+      <nav>
+        <a href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="https://tammylfisher.github.io/portfolio-website/"
+          >Portfolio</a
+        >
+        <a href="contact.html">Contact</a>
+      </nav>
+    </header>
+    <section class="content">
+      <div class="intro">
+        <img class="headerimg" src="img/header1.jpeg" />
+        <h1>Tammy L. Fisher RVT/Web Dev</h1>
+        <div class="Address">
+          <h5>
+            <span id="p1">Address:</span>1059 S. Manhattan Ave. Manhattan, KS.
+            66502 <span id="p1">Cell:</span>785-410-1549
+            <span id="p1">Email:</span>kochira_heruko1@yahoo.com
+          </h5>
+        </div>
+      </div>
+      <section>
```

Comment:
I think a little white space would improve readability.  That way it's easily visible what is a section/block